### PR TITLE
DOCS-#0000: fix pip install command for macos

### PR DESCRIPTION
**Pull request by MSHADroo**
## What do these changes do?

- [x] first commit message and PR title follow format outlined [here](https://modin.readthedocs.io/en/latest/development/contributing.html#commit-message-formatting)
  > **_NOTE:_**  If you edit the PR title to match this format, you need to add another commit (even if it's empty) or amend your last commit for the CI job that checks the PR title to pick up the new PR title.
- [ ] passes `flake8 modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [ ] passes `black --check modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [ ] signed commit with `git commit -s` 
- [ ] Resolves #? 
- [ ] tests added and passing
- [ ] module layout described at `docs/development/architecture.rst` is up-to-date

### CODE REVIEW

**Comment by YarShev** [comment]
@MSHADroo, thanks for fixing this. Could you please create an issue with this regard and link it to the PR in order to follow our [contributing](https://modin.readthedocs.io/en/stable/development/contributing.html) guide? If you have some problems during the process, please let me know. I'll be happy to help.

**Comment by anmyachev** [comment]
@mvashishtha you are more familiar with macos. Is that really how it works?

**Comment by pyrito** [comment]
@anmyachev I can confirm that quotes are needed for zsh. If you just do `pip install modin[ray]`, zsh complains saying `zsh: no matches found: modin[ray]`.

**Comment by anmyachev** [comment]
> @anmyachev I can confirm that quotes are needed for zsh. If you just do `pip install modin[ray]`, zsh complains saying `zsh: no matches found: modin[ray]`.

Thanks @pyrito!

File: README.md [hunk 1]
```diff
@@ -56,15 +56,18 @@ The charts below show the speedup you get by replacing pandas with Modin based o
 Modin can be installed with `pip` on Linux, Windows and MacOS:
 
 ```bash
-pip install modin[all] # (Recommended) Install Modin with all of Modin's currently supported engines.
+pip install "modin[all]" # (Recommended) Install Modin with all of Modin's 
+currently 
+supported engines.
```
Comment (anmyachev) [inline README.md:61]:
```suggestion
pip install "modin[all]" # (Recommended) Install Modin with all of Modin's currently supported engines.
```

File: README.md [hunk 2]
```diff
@@ -56,15 +56,18 @@ The charts below show the speedup you get by replacing pandas with Modin based o
...
 If you want to install Modin with a specific engine, we recommend:
 
 ```bash
-pip install modin[ray] # Install Modin dependencies and Ray.
-pip install modin[dask] # Install Modin dependencies and Dask.
-pip install modin[unidist] # Install Modin dependencies and Unidist to run on Unidist
+pip install "modin[ray]" # Install Modin dependencies and Ray.
+pip install "modin[dask]" # Install Modin dependencies and Dask.
+pip install "modin[unidist]" # Install Modin dependencies and Unidist to 
+run on Unidist
```
Comment (anmyachev) [inline README.md:70]:
```suggestion
pip install "modin[unidist]" # Install Modin dependencies and Unidist to run on Unidist
```

File: README.md [hunk 3]
```diff
@@ -56,15 +56,15 @@ The charts below show the speedup you get by replacing pandas with Modin based o
...
 
 If you want to install Modin with a specific engine, we recommend:
 
 ```bash
-pip install modin[ray] # Install Modin dependencies and Ray.
-pip install modin[dask] # Install Modin dependencies and Dask.
-pip install modin[unidist] # Install Modin dependencies and Unidist to run on Unidist
+pip install "modin[ray]" # Install Modin dependencies and Ray.
+pip install "modin[dask]" # Install Modin dependencies and Dask.
+pip install "modin[unidist]" # Install Modin dependencies and Unidist to run on Unidist
```
Comment (anmyachev) [inline README.md:67]:
```suggestion
pip install "modin[unidist]" # Install Modin dependencies and Unidist.
```

**Review by anmyachev — APPROVED** [review]
Thanks @MSHADroo!

**Comment by anmyachev** [comment]
@MSHADroo in order for us to merge your pull request, you need at least sign your commits (via `git commit -s`). Could you do it?

**Review by YarShev — COMMENTED** [review]
Is the readme the only place where we should fix the commands?

**Comment by anmyachev** [comment]
> Is the readme the only place where we should fix the commands?

I corrected the rest of the places.

**Review by YarShev — APPROVED** [review]

**Comment by anmyachev** [comment]
Thanks @MSHADroo!

**Pull request merged**